### PR TITLE
added a file with add func

### PR DESCRIPTION
**Pull request by deepakg-sacumen**

### CODE REVIEW

File: src/deepak.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def add(a,b):
```
Comment (vinodkumar-sacumen) [inline src/deepak.py:1]: Please add docstring at the First line of the file. And also add doc string to the function.

Comment (vinodkumar-sacumen) [inline src/deepak.py:1]: Add type hinting to the function parameters

**Review by vinodkumar-sacumen — CHANGES_REQUESTED** [review]
Please address the comments on the PR and also share the same with Ganesh and Guru

**Review by deepakg-sacumen — COMMENTED** [review]
I have updated the file by adding appropriate docstrings and added type hinting.

**Pull request merged**